resources_pages/install_ds_stack_ubuntu.md

```diff
@@ -529,20 +540,27 @@ This requires the `pyppeteer` package,
 which we can install as follows.
 
 ```bash
-conda install pyppeteer=0.2.2
+conda install pyppeteer
 pyppeteer-install
 ```
 
 Try this by going to `File -> Export notebook as... -> Export Notebook to WebPDF`.
+This may take a long time
```

Comment:
This is not working for me currently. I have opened https://github.com/jupyter/nbconvert/issues/1834